### DISCUSSION TITLE
Paste as verbatim shortcut

### DISCUSSION THREAD

**Discussion by TwiggyWan** [zim-desktop-wiki/zim-desktop-wiki#2664] [Ideas]
Hello, AFAIK there is no shortcut for 'right click > paste as verbatim' and I did not find it in the keyboard shortcuts setting window.

Would that be suitable for a (good first) issue? I use that a lot instead of the code block plugin since Find works inside verbatim blocks and not inside code blocks.

**Comment by brucewestfall** [comment]
I thought ctrl+shift+v did that.

**Reply by TwiggyWan** [reply]
It doesnt work on my setup (debian 12, xfce, zim 0.75.2)

further if i do ctrl t then ctrl v : 
![image](https://github.com/user-attachments/assets/11e142d1-21e2-4354-82c2-77bdf9796051)

which is different than : 
![image](https://github.com/user-attachments/assets/f70bfcee-4d56-4343-8e6d-3c48066831a0)

**Comment by brucewestfall** [comment]
mine is at `<Actions>/Paste FromHTML...etc.`

Also linux with XFCE and Zim 0.75.2

**Reply by TwiggyWan** [reply]
do you mean in the keybindings list?

![image](https://github.com/user-attachments/assets/58d23525-060c-4ab7-9baf-2e044e605397)

**Comment by brucewestfall** [comment]
I have the "Paste from HTML" plugin.  Not sure if it is default or from github.  That must be where the shortcut came from.  I use it all the time which is why i was so quick to respond.

Yup. Github.  Here it is  - https://github.com/cetinkaya/pastefromhtml

**Reply by TwiggyWan** [reply]
Thanks, I did not know this plugin.
It is nice but not for my use cases : pasting terminal output to a verbatim block, pasting non html stuff from different apps like teams or slack.
